source/_integrations/neurio_energy.markdown

```diff
@@ -39,7 +39,7 @@ api_secret:
   required: true
   type: string
 sensor_id:
-  description: "The sensor ID, a hex number as shown on the [PWRView webpage](https://mypwrview.generac.com/#settings/sensors), e.g., `0x0000XXXXXXXXXXXX`."
+  description: "The sensor ID, a hex number as shown on the [PWRView webpage](https://mypwrview.generac.com/#settings/sensors), e.g., `0x0000XXXXXXXXXXXX`.  The value must be in double quotes so it is kept a string instead of being interpreted as a number."
```

Comment:
This is not specific to this integration, but part of YAML.
We don't report this with every field we have. Please revert this part.